I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 18, in script
    $ count += 1
  File "game/script.rpy", line 18, in <module>
    $ count += 1
NameError: name 'count' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 18, in script
    $ count += 1
  File "/Users/nikolayhutsal/Library/renpy-7.3.5-sdk/renpy/ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/nikolayhutsal/Library/renpy-7.3.5-sdk/renpy/python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 18, in <module>
    $ count += 1
NameError: name 'count' is not defined

Darwin-19.0.0-x86_64-i386-64bit
Ren'Py 7.3.5.606
the_novel 1.0
Mon Apr 27 17:41:52 2020
